app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,24 @@
+class SessionsController < ApplicationController
+
+  skip_before_action :authorize
+
+  def new
+  end
+
+  def create
+    user = User.find_by(email: params[:email])
+    if user && user.authenticate(params[:password])
+      session[:user_id] = user.id
```

Comment:
The default Rails cookie store is EncryptedCookieStore. The session hash is serialized and set as the cookie’s value. An encrypted cookie is used so that the recipient can’t see the stored values. (Prior to Rails 4 CookieStore was used, which worked the same way, but wasn’t encrypted so stored values were (with a bit of effort) visible.